kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/importer/sink/PipelineDataSourceSink.java

```diff
@@ -259,8 +258,7 @@ private void executeUpdate(final Connection connection, final DataRecord dataRec
     
     private void executeBatchDelete(final Connection connection, final List<DataRecord> dataRecords) throws SQLException {
         DataRecord dataRecord = dataRecords.get(0);
-        String deleteSQL = importSQLBuilder.buildDeleteSQL(getSchemaName(dataRecord.getTableName()), dataRecord,
-                RecordUtils.extractConditionColumns(dataRecord, importerConfig.getShardingColumns(dataRecord.getTableName())));
+        String deleteSQL = importSQLBuilder.buildDeleteSQL(getSchemaName(dataRecord.getTableName()), dataRecord, RecordUtils.extractConditionColumns(dataRecord, importerConfig.getShardingColumns(dataRecord.getTableName())));
```

Comment:
1, Existing code could be kept without modification.

2, There's error in CheckStyle: `Line is longer than 200 characters (found 224). (261:0) [LineLength]`

Reply:
Sorry, looks like I need to know more about this knowledge.Thank u patient reply.